rules/defaults.yaml

```diff
@@ -2443,3 +2443,67 @@
 		RequiresCondition: overload_active
 	Sellable:
 		RequiresCondition: !overload_active
+
+####Superweapons####
+
+DummyShakerVision:
```

Comment:
Note that I am in the process of having a RevealShroudWarhead - https://github.com/AttacqueSuperior/Engine/pull/13 - so I'd like to see atleast a TODO: somewhere that this will get replaced in the semi-short term.